src/api/spec/controllers/webui/cloud/ec2/upload_job/logs_controller_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe Webui::Cloud::UploadJob::LogsController, type: :controller, vcr: true do
+  let!(:user) { create(:confirmed_user, login: 'tom') }
+  let!(:ec2_configuration) { create(:ec2_configuration, user: user) }
```

Comment:
I would move the user with and user without ev2 config to the respective blocks. Otherwise you go back and forth with [removing](v) the config again.